js/apps/admin-ui/public/locales/pt-BR/authentication.json

```diff
@@ -18,9 +18,9 @@
     "none": "Nenhum"
   },
   "flow": {
-    "browserFlow": "Fluxo de browser",
-    "registrationFlow": "Fluxo de registro",
-    "directGrantFlow": "Fluxo de Direct Grant"
+    "browser": "Fluxo de browser",
+    "registration": "Fluxo de registro",
+    "direct grant": "Fluxo de Direct Grant"
```

Comment:
we should also add these but translated  "reset credentials": "Reset credentials flow", "clients": "Client authentication flow", "docker auth": "Docker authentication flow"